it/controllers/employment/EmploymentDetailsAndBenefitsControllerISpec.scala

```diff
@@ -0,0 +1,472 @@
+
+  object ContentEN {
+    val h1Expected = "maggie"
+    val titleExpected = "Employment details and benefits"
+    val captionExpected = s"Employment for 6 April ${taxYear - 1} to 5 April $taxYear"
+    val p1ExpectedAgent = "You cannot update your client’s employment information until 6 April 2022."
+    val p1ExpectedIndividual = "You cannot update your employment information until 6 April 2022."
+
+    val fieldNames = List("Employment details",
+      "Benefits")
+
+    val buttonText = "Return to employment summary"
+  }
+
+  object ContentCY {
+    val h1Expected = "maggie"
+    val titleExpected= "Employment details and benefits"
+    val captionExpected = s"Employment for 6 April ${taxYear - 1} to 5 April $taxYear"
+    val p1ExpectedAgent = "You cannot update your client’s employment information until 6 April 2022."
+    val p1ExpectedIndividual = "You cannot update your employment information until 6 April 2022."
+
+    val fieldNames = List("Employment details",
+      "Benefits")
+
+    val buttonText = "Return to employment summary"
+  }
+
+  val amount: BigDecimal = 100
+
+  val BenefitsModel: Benefits = Benefits(
+    Some(amount), Some(amount), Some(amount), Some(amount), Some(amount), Some(amount), Some(amount), Some(amount), Some(amount), Some(amount),
+    Some(amount), Some(amount), Some(amount), Some(amount), Some(amount), Some(amount), Some(amount), Some(amount), Some(amount), Some(amount),
+    Some(amount), Some(amount), Some(amount), Some(amount), Some(amount), Some(amount), Some(amount), Some(amount)
+  )
+
+  def fullModel(benefitsModel:Option[EmploymentBenefits]):AllEmploymentData = {
+      AllEmploymentData(
+      hmrcEmploymentData = Seq(
+        EmploymentSource(
+          employmentId = "001",
+          employerName = "maggie",
+          employerRef = Some("223/AB12399"),
+          payrollId = Some("123456789999"),
+          startDate = Some("2019-04-21"),
+          cessationDate = Some("2020-03-11"),
+          dateIgnored = Some("2020-04-04T01:01:01Z"),
+          submittedOn = Some("2020-01-04T05:01:01Z"),
+          employmentData = Some(EmploymentData(
+            submittedOn = ("2020-02-12"),
+            employmentSequenceNumber = Some("123456789999"),
+            companyDirector = Some(true),
+            closeCompany = Some(false),
+            directorshipCeasedDate = Some("2020-02-12"),
+            occPen = Some(false),
+            disguisedRemuneration = Some(false),
+            pay = Pay(34234.15, 6782.92, Some(67676), "CALENDAR MONTHLY", "2020-04-23", Some(32), Some(2))
+          )),
+          employmentBenefits = benefitsModel
+        )
+      ),
+      hmrcExpenses = None,
+      customerEmploymentData = Seq(),
+      customerExpenses = None
+    )
+  }
+
+
+  "as an individual in English" when {
+
+    ".show" should {
+
+      "return an action with benefits status Cannot update when no benefits data is in session" which {
```

Comment:
can you write what that action is e.g. render the page where the status for benefits is "cannot update"